source/frontend/src/features/properties/map/acquisition/detail/AcquisitionSummaryView.tsx

```diff
@@ -23,13 +23,26 @@ const AcquisitionSummaryView: React.FunctionComponent<
   const keycloak = useKeycloakWrapper();
   const detail: DetailAcquisitionFile = DetailAcquisitionFile.fromApi(acquisitionFile);
 
+  const projectName =
+    acquisitionFile?.project !== undefined
+      ? acquisitionFile?.project?.code + ' - ' + acquisitionFile?.project?.description
```

Comment:
hmm, if the code or description is null/undefined will that get written out here as a string? hmm... nvm they are both required fields in the db.

Mild nit I guess since this only works if the frontend knows these fields are non-nullable.